views.py

```diff
@@ -1332,6 +1332,7 @@ def typesetting_preview_galley(
         'identifier_type': 'id',
         'identifier': article.pk if article else proofing_task.round.article.pk,
         'article_content': galley.file_content(),
+        'tables_in_galley': journal_logic.get_all_tables_from_html(galley.file_content()),
```

Comment:
Can we avoid calling `galley.file_content` twice? we have to read the whole file from disk twice.